ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -0,0 +1,44 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.5"
+  namespace: ocp
+test_binary_build_commands: make test
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-test
+      commands: ./bin/extended-platform-tests run openshift/conformance/serial --run
+        "\[Serial\] olm version should contain the source commit id"
+      from: bin
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - as: e2e-cmd
+      commands: ./bin/extended-platform-tests run openshift/test-cmd
+      from: bin
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: ipi-aws
```

Comment:
One note here: setup like this you have created a single job, where the test section consists of two separate steps, run sequentially against a single provisioned cluster. This may make sense if you are sure that the first step does not break the cluster. But because previously you had *two* jobs, I thought I will point it out. If you still want to have two jobs, then the YAML will look like something like this:

```yaml
tests:
- as: e2e-aws:
  steps:
    ...
    test:
    - as: e2e-test
      ...
    workflow: ipi-aws
- as: e2e-cmd:
  steps:
    ...
    test:
    - as: e2e-cmdk
      ...
    workflow: ipi-aws
```

Note the diference between having a single job with two test steps, and having two jobs, having a single test step each.

Reply:
@petr-muller Thanks for your reminder. Yes, I'm sure the previous step won't break the following step. The reason why I use a single job is that I thought a single job will run the above`make build` command only once. If I use two jobs, I guess the `make build` command will be run twice. Is it? Or am I misunderstand?

Reply:
`make build` will be run only once. ci-operator will run the commands in `binary_build_commands` and create a `bin` image, which is reused by all later steps that use it with `from: bin`.

The main difference between a job that runs two tests and two jobs that run one test each is that the former installs one cluster and then runs the tests sequentially, and the latter installs a cluster *each* and run tests in parallel.

Reply:
@petr-muller Thank you so much! It's clear to me now.